ActindoWorkFlow/Overview/03_Features.md

```diff
@@ -0,0 +1,27 @@
+# Key features
+
+Model your specific workflows:
+
+- Process Orchestration      
+  The *Workflows* module enables rapid modeling, deployment and monitoring of business processes to automate your workflows.
+
+- Business Rule Management      
+  The *Workflows* module allows you to design and execute business rules. This allows you to cast decision paths for specific processes and scenarios and easily automate them.
+
+- Live Data and Log     
+  In the Workflow Manager, all processes and data can be viewed live, and a detailed log is created for each process. This means that you can always see where an order or return is currently located, and can quickly intervene in the event of errors and track all steps.
+
+- Individual Workflows     
+  Maximum flexibility in automating your processes: All workflows are defined by you and can be fully customized. This means that your specific business processes can be mapped precisely.
+
+
+## Efficient workflow management
+
+Satisfied customers thanks to smooth processes   
+In the course of digitalization, new business processes are constantly emerging that need to be introduced and monitored. The *Workflows* module supports you in collecting and mapping the multitude of separate processes in your company in one central location. You define the rules according to which your workflows run and can thus continuously optimize them. Monitoring and troubleshooting can also be organized efficiently in this way. Smooth processes in all business areas ultimately lead to faster processing of orders or returns and thus to a better customer experience.
+
+
+## The duo for successful digitization
+
+Simple networking - internally and externally      
```

Comment:
Is this a sentence or a headline? Needs a full stop in the end or some more markup